src/jobs/districtCorona/index.js

```diff
@@ -1,56 +1,133 @@
-		if (stats.timeLine.totalCases < 1.5 * lastSavedStats.timeLine.totalCases) {
-			await DistrictCoronaDbService.saveDistrictStats(stats)
+		let totalNewCases = 0
+		todayCasesbyDistricts.data && todayCasesbyDistricts.data.forEach((district)=>{
+			totalNewCases+=parseInt(district["Value"])
+		})
+		
+		let allOutcomes = await axios.get(`http://portal.edcd.gov.np/rest/api/fetch?filter=outcomeBetween&type=dayByDay&eDate=${today}&disease=COVID-19`)
+		if(allOutcomes.data.length==0){
+			allOutcomes = await axios.get(`http://portal.edcd.gov.np/rest/api/fetch?filter=outcomeBetween&type=dayByDay&eDate=${yesterday}&disease=COVID-19`)
+		}
+
+		let totalDeaths = 0
+		let totalRecovered = 0
+		allOutcomes.data && allOutcomes.data.forEach(district=>{
+			totalDeaths+=parseInt(district["Number of deaths"])
+			totalRecovered+=parseInt(district["Number of cases recovered"])
+		})
+
+		let allCases = await axios.get(`https://portal.edcd.gov.np/rest/api/fetch?filter=casesBetween&type=dayByDay&eDate=${today}&disease=COVID-19`,{httpsAgent})
+		if(allCases.data.length==0){
+			allCases = await axios.get(`https://portal.edcd.gov.np/rest/api/fetch?filter=casesBetween&type=dayByDay&eDate=${yesterday}&disease=COVID-19`,{httpsAgent})
 		}
-	} else {
+
+		let totalCases = 0
+		allCases.data.forEach(district=>{
+			totalCases+=parseInt(district["Value"])
+		})
+
+		let coronaTimeLine = {
+			totalCases,
+			newCases: totalNewCases,
+			totalRecoveries: totalRecovered,
+			newRecoveries: totalNewRecovered,
+			totalDeaths,
+			newDeaths: totalNewDeaths
+		}
+
+		const districts = await axios.get('https://data.nepalcorona.info/api/v1/districts')
+
+		const districtCoronaStats = await axios.get('https://data.nepalcorona.info/api/v1/covid/summary')
+
+		const districtMetrics = []
+
+		districts.data.forEach((district) => {
+			const totalCases = districtCoronaStats.data.district.cases.filter((x) => x.district === district.id)[0]
+			const activeCases = districtCoronaStats.data.district.active.filter((x) => x.district === district.id)[0]
+			const recovered = districtCoronaStats.data.district.recovered.filter((x) => x.district === district.id)[0]
+			const deaths = districtCoronaStats.data.district.deaths.filter((x) => x.district === district.id)[0]
+			const districtMetric = {
+				name: district.title,
+				nepaliName: district.title_ne,
+				totalCases: (totalCases && totalCases.count && totalCases.count) || 0,
+				activeCases: (activeCases && activeCases.count && activeCases.count) || 0,
+				recovered: (recovered && recovered.count && recovered.count) || 0,
+				deaths: (deaths && deaths.count && deaths.count) || 0,
+			}
+
+			districtMetrics.push(districtMetric)
+		})
+
+		const stats = {
+			createdDate: new Date(),
+			timeLine: coronaTimeLine,
+			districts: districtMetrics,
+			source: 'nepalcorona.info',
+		}
+
 		await DistrictCoronaDbService.saveDistrictStats(stats)
+
+		return stats
+
+	}catch(error){
+		console.log("error in national corona stats ",error)
```

Comment:
can you use logger instead of console for logging? 
Here we can use `logger.error`
It should print in console as well as can be used to filter logs in logentries in Heroku.